pre-commit.sh

```diff
@@ -133,7 +133,7 @@ done
 
 header test_raw_api
 
-if TEST_ACCOUNT_ID="$(head -n 1 ~/.b2_auth)" TEST_APPLICATION_KEY="$(tail -n 1 ~/.b2_auth)" python -m b2sdk.raw_api
+if TEST_APPLICATION_KEY_ID="$(head -n 1 ~/.b2_auth)" TEST_APPLICATION_KEY="$(tail -n 1 ~/.b2_auth)" python -m b2sdk.raw_api
```

Comment:
@bwbeach I think I asked this last time, do we need to include using `TEST_ACCOUNT_ID` somewhere?
Please show me how we build the python stuff we were talking about and I can add it.

Reply:
since tests have passed in Travis on this PR, it must mean @bwbeach has already adjusted the CI settings